emission/individual_tests/TestOverpass.py

```diff
@@ -44,19 +52,171 @@ def test_overpass(self):
 
     #Test utilizes the functions get_stops_near, get_public_transit_stops, and make_request_and_catch.  
     def test_get_stops_near(self):
-        actual_result = enetm.get_stops_near(loc1, 150.0)[0]['routes'][0]['tags']
-        expected_result = {'from': 'National Renewable Energy Lab', 'name': 'RTD Route 125: Red Rocks College', 'network': 'RTD', 'network:wikidata': 'Q7309183', 'network:wikipedia': 'en:Regional Transportation District', 'operator': 'Regional Transportation District', 'public_transport:version': '1', 'ref': '125', 'route': 'bus', 'to': 'Red Rocks College', 'type': 'route'}
-        self.assertEqual(expected_result, actual_result)
+        stops_at_loc1 = enetm.get_stops_near([loc1['coordinates']], 150.0)[0]
+        first_stop_routes = stops_at_loc1[0]['routes']
+        self.assertEqual(len(first_stop_routes), 2)
+        first_stop_first_route_tags = first_stop_routes[0]['tags']
+        expected_tags = {'from': 'National Renewable Energy Lab', 'name': 'RTD Route 125: Red Rocks College', 'network': 'RTD', 'network:wikidata': 'Q7309183', 'network:wikipedia': 'en:Regional Transportation District', 'operator': 'Regional Transportation District', 'public_transport:version': '1', 'ref': '125', 'route': 'bus', 'to': 'Red Rocks College', 'type': 'route'}
+        self.assertEqual(first_stop_first_route_tags, expected_tags)
    
     #Get_stops_near generates two stops from the given coordinates.
     # Get_predicted_transit_mode finds a common route between them (train).
     def test_get_predicted_transit_mode(self):
-        stop1 = enetm.get_stops_near(loc2, 400.0)
-        stop2 = enetm.get_stops_near(loc3, 400.0)
-        actual_result = enetm.get_predicted_transit_mode(stop1, stop2)
+        [stops_at_loc2, stops_at_loc3] = enetm.get_stops_near(
+            [loc2['coordinates'], loc3['coordinates']],
+            400.0,
+        )
+        actual_result = enetm.get_predicted_transit_mode(stops_at_loc2, stops_at_loc3)
         expected_result = ['train', 'train']
         self.assertEqual(actual_result, expected_result)
 
+    def test_chunk_list(self):
+        # Case 1: List of 10 elements with chunk size of 3.
+        data = list(range(1, 11))  # [1, 2, ..., 10]
+        chunk_size = 3
+        chunks = list(enetm.chunk_list(data, chunk_size))
+        expected_chunks = [[1, 2, 3], [4, 5, 6], [7, 8, 9], [10]]
+        self.assertEqual(chunks, expected_chunks)
+
+        # Case 2: Exact division
+        data_exact = list(range(1, 10))  # [1, 2, ..., 9]
+        chunk_size = 3
+        chunks_exact = list(enetm.chunk_list(data_exact, chunk_size))
+        expected_chunks_exact = [[1, 2, 3], [4, 5, 6], [7, 8, 9]]
+        self.assertEqual(chunks_exact, expected_chunks_exact)
+
+        # Case 3: Empty list
+        data_empty = []
+        chunks_empty = list(enetm.chunk_list(data_empty, chunk_size))
+        self.assertEqual(chunks_empty, [])
```

Comment:
I don't see much value in testing this function by itself.
Validate the behavior of `get_public_transit_stops` and that will tell us whether the chunking is working correctly